test/fcntl/test_fcntl_open.c

```diff
@@ -151,8 +151,18 @@ void test() {
   errno = 0;
 }
 
+void test_open_create_no_permissions() {
+  int res = open("a", O_CREAT, 0);
+  printf("error: %s\n", strerror(errno));
+  assert(res >= 0);
+  struct stat st;
+  assert(stat("a", &st) == 0);
+  assert((st.st_mode & 0777) == 0);
```

Comment:
Is this the line that fails on windows?  Perhaps we could make it more flexible?

As it stands if we add `@no_windows` the whole test will no longer run on windows.  Alternatively you could do this:

```
if self.getsetting('NODERAWFS') and WINDOWS:
   self.skipTest(...)
```

That way only that particular variant will be skipped.

Reply:
Yeah, that's the one: See https://logs.chromium.org/logs/emscripten-releases/buildbucket/cr-buildbucket/8728106183896305361/+/u/Emscripten_testsuite__core0_/stdout (line 19 of the original)

Reply:
But it also fails on nodefs, not just on noderawfs.

Reply:
We have a bunch of test that have something like `if nodefs and WINDOWS:  skip`.. so we can do that here to maybe?